src/Weft/Internal/Types.hs

```diff
@@ -204,16 +182,17 @@ data None (ts :: TypeState) =
 deriving instance AllHave Eq (None ts) => Eq (None ts)
 deriving instance AllHave Show (None ts) => Show (None ts)
 
-data Gql (q :: TypeState -> *)
-         (m :: TypeState -> *)
+data Gql (q :: *)
```

Comment:
this is great, I was just trying to fix a bunch of errors in the server from when this was `TypeState -> *`